.github/workflows/publish-commit-bottles.yml

```diff
@@ -0,0 +1,62 @@
+name: Publish and commit bottles
+
+on: repository_dispatch
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    container:
+      image: homebrew/brew
+    env:
+      HOMEBREW_DEVELOPER: 1
+      HOMEBREW_FORCE_HOMEBREW_ON_LINUX: 1
+      HOMEBREW_GIT_EMAIL: homebrew-test-bot@lists.sfconservancy.org
```

Comment:
This is set by test-bot already, right?

Reply:
Looks like it is. This was copied over from Jenkins configuration. I'll take it out.